Source/PeripheralManager.swift

```diff
@@ -2,11 +2,11 @@ import Foundation
 import CoreBluetooth
 import RxSwift
 
-/// PeripheralManager is a class implementing ReactiveX API which wraps all Core Bluetooth Peripheral's functions allowing to
-/// advertising, publishing L2CAP channels and more.
-/// You can start using this class by add services and starting advertising.
+/// PeripheralManager is a class implementing ReactiveX API which wraps all the Core Bluetooth Peripheral's functions, that allow to
+/// advertise, to publish L2CAP channels and more.
```

Comment:
This is something we missed out before. Should be: `(...), advertises, publishes to L2CAP channels, and more`